clock/test_clock_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	"time"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
nit: I think we could also change all `assert` calls to `require` below.

Reply:
Done!